src/courtroom.cpp

```diff
@@ -2577,7 +2577,7 @@ void Courtroom::log_ic_text(QString p_name, QString p_showname,
 {
   chatlogpiece log_entry(p_name, p_showname, p_message, p_action, p_color);
   ic_chatlog_history.append(log_entry);
-  if (ao_app->get_auto_logging_enabled())
+  if (ao_app->get_auto_logging_enabled() && !ao_app->log_filename.contains("/Demo playback/"))
```

Comment:
I didn't realize this copypasting BS existed with `get_auto_logging_enabled`. Let's try not to make it worse.